src/extensions/extension_manager/inc/aduc/extension_manager_download_options.h

```diff
@@ -10,4 +12,8 @@ typedef struct tagADUC_ExtensionManager_Download_Options
         timeoutInMinutes; /**< The maximum number of minutes the content downloader should wait for the download to complete(whilst the network interface stays up). */
 } ExtensionManager_Download_Options;
 
+EXTERN_C_BEGIN
+extern ExtensionManager_Download_Options Default_ExtensionManager_Download_Options;
```

Comment:
why is this marked as "extern"? Is the definition in external linkage (e.g. a separate module)?

Reply:
Yes, the definition is in extension_manager_helper.cpp.  It used to be in extension_manager.cpp, but that lead to a duplicate, conflicting definition because extension_manager.cpp includes extension_manager_download_options.h.